vaadin-split-layout-flow-parent/vaadin-split-layout-flow/src/main/java/com/vaadin/flow/component/splitlayout/SplitLayout.java

```diff
@@ -182,6 +182,35 @@ public SplitLayout(Component primaryComponent,
         addToSecondary(secondaryComponent);
     }
 
+    /**
+     * Constructs a VaadinSplitLayout with the orientation.
+     *
+     * @param orientation
+     *            the orientation set to the layout
+     */
+    public SplitLayout(Orientation orientation) {
+        setOrientation(orientation);
+        addAttachListener(
+                e -> this.requestStylesUpdatesForSplitterPosition(e.getUI()));
+    }
+
+    /**
+     * Constructs a VaadinSplitLayout with the given initial components to set
+     * to the primary and secondary splits and with the orientation.
+     *
+     * @param primaryComponent
+     *            the component set to the primary split
+     * @param secondaryComponent
+     *            the component set to the secondary split
+     * @param orientation
+     *            the orientation set to the layout
+     */
+    public SplitLayout(Component primaryComponent, Component secondaryComponent,
+            Orientation orientation) {
+        this(primaryComponent, secondaryComponent);
+        setOrientation(orientation);
```

Comment:
We probably need to call `addAttachListener` here too, as in other constructors with `setOrientation`.